frontend/src/pages/ApprovedActivityReport/index.js

```diff
@@ -0,0 +1,391 @@
+      if (report.activityRecipients.length > 1) {
+        recipientTypeLabel = `${recipientTypeLabel}s`;
+      }
+      setRecipientType(recipientTypeLabel);
+      const arRecipients = report.activityRecipients.map((arRecipient) => arRecipient.name).join(', ');
+      setRecipients(arRecipients);
+      setDisplayId(report.displayId);
+      setCreator(report.author.fullName);
+      setCollaborators(report.collaborators);
+      setApprovingManagers(report.approvingManager.fullName);
+      setAttendees(report.participants.join(', '));
+      const newCount = report.numberOfParticipants.toString();
+      setParticipantCount(newCount);
+      setReasons(report.reason.join(', '));
+      setProgramType(report.programTypes.join(', '));
+      setStartDate(moment(report.startDate, 'MM/DD/YYYY').format('MMMM D, YYYY'));
+      setEndDate(moment(report.endDate, 'MM/DD/YYYY').format('MMMM D, YYYY'));
+      setDuration(`${report.duration} hours`);
+      setMethod(formatMethod(report.ttaType, report.virtualDeliveryType));
+      setRequester(formatRequester(report.requester));
+
+      // second table
+      setTopics(report.topics.join(', '));
+      setECLKCResources(createResourceMarkup(report.ECLKCResourcesUsed));
+      setNonECLKCResourcesUsed(createResourceMarkup(report.nonECLKCResourcesUsed));
+      setAttachments(mapAttachments(report.attachments));
+
+      // // third table
+      setContext(report.context);
+      const [goalHeadings, goals] = calculateGoalsAndObjectives(report);
+      setGoalsAndObjectiveHeadings(goalHeadings);
+      setGoalsAndObjectives(goals);
+
+      // fourth table
+      setManagerNotes(report.managerNotes);
+      setCreatorNotes(report.additionalNotes);
+    }).catch((err) => {
+      // eslint-disable-next-line no-console
+      console.log(err);
+      setSomethingWentWrong(true);
+    });
+  }, [match.params.activityReportId, user]);
+
+  async function handleCopyUrl() {
+    try {
+      await navigator.clipboard.writeText(window.location.href);
+      setSuccessfullyCopiedClipboard(true);
+    } catch (err) {
+      // eslint-disable-next-line no-console
+      console.error(err);
+      setSomethingWentWrongWithClipboard(true);
+    }
+  }
+
+  if (notAuthorized) {
+    return (
+      <div className="usa-alert usa-alert--error no-print" role="alert">
+        <div className="usa-alert__body">
+          <h4 className="usa-alert__heading">Unauthorized</h4>
+          <p className="usa-alert__text">
+            Sorry, you are not allowed to view this report
+          </p>
+        </div>
+      </div>
+    );
+  }
+
+  if (somethingWentWrong) {
+    return (
+      <div className="usa-alert usa-alert--warning no-print">
+        <div className="usa-alert__body">
+          <p className="usa-alert__text">
+            Sorry, something went wrong.
+          </p>
+        </div>
+      </div>
+    );
+  }
+
+  return (
+    <>
+      <Helmet>
+        <title>
+          Activity Report
```

Comment:
Can we make this title match the format requested in [TTAHUB-190](https://ocio-jira.acf.hhs.gov/browse/TTAHUB-190)?